extensions/OWNERS

```diff
@@ -5,15 +5,15 @@
   rules: [
     {
       owners: [
-        {name: 'ampproject/wg-ui-and-a11y'},
+        {name: 'ampproject/wg-components'},
         {name: 'aghassemi', requestReviews: false},
       ],
     },
     {
       pattern: '**/*-player.{js,md}',
       owners: [
         {name: 'alanorozco', notify: true},
-        {name: 'wassgha', notify: true},
+        {name: 'wassgha', notify: false},
```

Comment:
`notify: false` is default, so that can be removed (unless you meant `requestReviews: false`)